src/models/tools/geometry/jxg-movable-line.ts

```diff
@@ -88,12 +93,27 @@ export const movableLineChangeAgent: JXGChangeAgent = {
           },
           ...overrides
         });
+
+      return [line, interceptPoint, slopePoint];
     }
   },
 
   // update can be handled generically
   update: objectChangeAgent.update,
 
-  // delete can be handled generically
-  delete: objectChangeAgent.delete
+  delete: (board, change) => {
+    if (!change.targetID) { return; }
```

Comment:
FYI, braces are not necessary on one-line if statements with our current lint settings.